githubdelivery/src/main/java/io/dockstore/githubdelivery/GithubDeliveryS3Client.java

```diff
@@ -179,32 +179,38 @@ private void submitGitHubDeliveryEventsByHour(String prefix, WorkflowsApi workfl
     private void submitGitHubDeliveryEventsByKey(String key, WorkflowsApi workflowsApi) {
         String deliveryid = key.split("/")[2]; //since key is in YYYY-MM-DD/HH/deliveryid format
         try {
-            String body = getObject(key);
-            JsonObject jsonObject = GSON.fromJson(body, JsonObject.class);
-            if (jsonObject.get("action").getAsString().equals("added") || jsonObject.get("action").getAsString().equals("removed")) {
-                InstallationRepositoriesPayload payload = getGitHubInstallationRepositoriesPayloadByKey(body, key);
+            String obj = getObject(key);
```

Comment:
Nit: it's not immediately clear to me what `obj` is given that there are `JsonObject`s below. Suggesting that the variable be renamed to `s3FileContent` or `s3GitHubObject`